src/hooks/useInitializeAddCardSDK.ts

```
@@ -89,7 +89,9 @@ const captureCard = (framesSDK, captureCardAction) => async () => {
         authorizationResponse
       );
     }
-  } catch (err: any) {}
+  } catch (err: any) {
+    console.error(err);
+  }
```

Moral of the story -- never swallow errors :)